### ISSUE TITLE
Can you please also support Waterfox browser ?

### ISSUE THREAD

**Issue by LL25255252** [JustOff/ublock0-updater#7]
My main browser is Pale Moon but sometimes I also use Waterfox.

Waterfox 55.2.2
Win 7 x64
Thanks in advance.

**Comment by JustOff** [comment]
Waterfox is supported, you just have to install Updater from GitHub, and not from Pale Moon add-ons site.

**Comment by LL25255252** [comment]
First thanks a lot for replying and sorry for my delayed response.
1. I'm almost sure I installed (6 days ago) "ublock-update" from https://github.com/JustOff/ublock0-updater/releases (and not from Pale Moon...)  but it didn't work (although there was a new uBO version it didn't "find" it) so I uninstalled the addon.
2. There's a new uBO  version (1,14,20) on AMO so today Waterfox addon's mananger itself found it (without the need of ublock0-updater.
3. I'll give it another shot...I'll install ublock0-update again and see what happens on the next uBO updates.

Waterfox 55.2.2
Win 7 x64
Thanks in advance.
